DbTables/inc/CalCosmicEnergyCalib.hh

```diff
@@ -37,7 +37,8 @@ namespace mu2e {
 
     CalCosmicEnergyCalib():DbTable(cxname,"cal.cosmicenergycalib","roid,EPeak,ErrEPeak,Width,ErrWidth,chisq"){}
 
-    const Row& row(std::uint16_t roid) const { return _rows.at(roid); }
+    const Row& row(CaloSiPMId  _roid) const {
```

Comment:
Sorry one more - this underscore is misleading

Reply:
fixed